addons/web/static/src/less/webclient.less

```diff
@@ -128,3 +128,18 @@ div.o_boolean_toggle {
 .bg-info-full {
     background-color: @brand-info;
 }
+
+.o_web_accesskey_overlay {
+    font-family: @font-family-sans-serif;
+    position: absolute;
+    width: 100%;
+    height: 100%;
+    left: 0;
+    top: 0;
+    z-index: 1000000;
+    background-color: rgba(0,0,0,.6);
+    color: #FFFFFF;
+    justify-content: center;
+    display: flex;
+    align-items: center;
+}
```

Comment:
missing empty line at the end (no idea why, but github displays this in red, so it should be fixed!